humblebundle_downloader/cli.py

```diff
@@ -73,6 +108,32 @@ def parse_args(args):
         help=("The purchase download key. Find in the url on the "
               "products/bundle download page. Can set multiple"),
     )
+    parser.add_argument(
+        '-b', '--write-buffer',
+        type=str, default=1024*1024,
```

Comment:
Why is this the only argument with a default set here while all the others are set in the download_library.py __init__?